File: osb-mobile-main/src/features/card/redux/models/request/BindUnnamedCardRequest.ts
import { ApiRequest } from '_config'
import { CardHolder } from '../CardHolder'
import { CardHolderContact } from '../CardHolderContact'
import { CardOwner } from '../CardOwner'
import { UsageType } from '../Enum'

export interface BindUnnamedCardRequest extends ApiRequest {
  identifierCard: string
  usageType: UsageType
  cardOwner: CardOwner
  cardHolder: CardHolder
  cardHolderContact: CardHolderContact
}

File: osb-core-main/Platform/Integration/Osb.Core.Platform.Integration.Entity/FitBank/Models/Request/BindUnnamedCardRequest.cs
using Osb.Core.Platform.Integration.Entity.Models.Request.Base;

namespace Osb.Core.Platform.Integration.Entity.FitBank.Models.Request
{
    public class BindUnnamedCardRequest : BaseRequest
    {
        public new string Method { get => "BindUnnamedCard"; }
        public CardOwnerRequest CardOwner { get; set; }
        public CardHolderRequest CardHolder { get; set; }
        public CardHolderContactRequest CardHolderContact { get; set; }
        public string IdentifierCard { get; set; }
        public string UsageType { get; set; }
    }
}